src/phoenix/config.py

```diff
@@ -53,6 +61,22 @@ def get_running_pid() -> Optional[int]:
 PORT = 6006
 # The prefix of datasets that are auto-assigned a name
 GENERATED_DATASET_NAME_PREFIX = "phoenix_dataset_"
+# The work directory for saving, loading, and exporting datasets
+WORKING_DIR = get_working_dir()
+
+try:
+    print(f"⚙️ Setting up a working directory at: {WORKING_DIR}")
+    print(f"ℹ️ To change, set the `{ENV_WORKING_DIR}` env var before importing phoenix.")
+    for path in (
+        ROOT_DIR := WORKING_DIR,
+        EXPORT_DIR := ROOT_DIR / "exports",
+        DATASET_DIR := ROOT_DIR / "datasets",
+        TRACE_DATASET_DIR := ROOT_DIR / "trace_datasets",
+    ):
+        path.mkdir(parents=True, exist_ok=True)
+except Exception:
+    print(f"⚠️ Failed to create a working directory at {WORKING_DIR}")
```

Comment:
maybe revise the wording a bit because the exception is probably something like "can't create subfolder" due to permission errors

Reply:
printing the `str(exception)` is probably also helpful

Reply:
Will do, in reality it's kinda hard to hit this since `exit_ok=True` but I agree, will soften the wording